examples/drush

```diff
@@ -1,13 +1,32 @@
 #!/usr/bin/env sh
```

Comment:
That's a good idea -- particularly since now, you can keep it named 'drush.wrapper' if using it with the drush finder.